pegasus/migrations/123_add_grade_displayweights_to_tutorials.rb

```diff
@@ -0,0 +1,9 @@
+Sequel.migration do
+  change do
+    alter_table(:tutorials) do
+      add_column :displayweight_k5, String
+      add_column :displayweight_middle, String
+      add_column :displayweight_high, String
+    end
+  end
+end
```

Comment:
Are we able to have the migration as a separate PR. Asher's recommendation was that if at all possible, migrations and code that uses migrations should deploy in separate DTPs (in case something goes wrong with the migration).

Reply:
I thought about it and convinced myself it wasn't necessary here, but you've convinced me back.  Will do :)

Reply:
Migration is here: https://github.com/code-dot-org/code-dot-org/pull/18654